toc/working-groups/foundational-infrastructure.md

```diff
@@ -325,6 +325,7 @@ areas:
   - cloudfoundry/bosh-workstation
   - cloudfoundry/bpm-release
   - cloudfoundry/bsdtar
+  - cloudfoundry/cnbapplifecycle
```

Comment:
I think the cnbapplifecycle repo should be added to the [App Runtime Platform WG](https://github.com/cloudfoundry/community/blob/main/toc/working-groups/app-runtime-platform.md), Diego area and not to Foundational Infrastructure.
At least, the other lifecycle implementation live in ARP/Diego.

Reply:
Sorry, if my comment on the RFC with the example wasn't fully clear but definitely the FI WG isn't the right place for it.